Dev docs: tell NVDA to treat source directory as app directory, allowing dev docs build with Sphinx to succeed

josephsl opened the pull request:
Hi,
Foundation to allow building source code docs locally and through services suc has Read The Docs:

### Link to issue number:
Fixes #11971 

### Summary of the issue:
When building source code documentation, Python will notice that globalvars.appDir is not found, causing Sphinx to give up when building docs.

### Description of how this pull request fixes the issue:
In Sphinx config file, Sphinx will be told to "define" source code directory as app directory, allowing dev docs build to succeed.

### Testing performed:
Tested with Sphinx 2.2.2 and 3.4.1 with dev docs builds succeeding.

### Known issues with pull request:
None, although Sphinx raises warnings, which can be handled in subsequent pull requests.

### Change log entry:
Changes for developers:

You can now build source code documentation locally using Sphinx. (#11971)

Thanks.

josephsl committed:
Dev docs/config: point globalVars.appDir to source directory when building source code dev docs. re #11971.

Before building dev docs with Sphinx, config module is imported without NVDA knowing where the app dir is i.e. globalVars.appDir is undefined. Therefore tell Sphinx that globalVars.appDir is source directory so Sphinx can build source code documentation.

josephsl committed:
Dev docs/Sphinx config: update copyright header

josephsl commented:
Hi,

NOt related to the PR, but I'm beginning to suspect that ARIA treegrid test is going to give an impression that the PR's didn't pass wen they are actually working as expected.

Thanks.

lukaszgo1 commented on devDocs/conf.py:
```diff
@@ -100,6 +100,11 @@ class AppArgs:
 
+# #11971: NVDA is not running, therefore app dir is undefined.
+# Therefore tell NVDA that apt source directory is app dir.
+import globalVars  # noqa: E402
```
`GlobalVars` is already imported above when setting `appArgs`. Also it makes sense to set `appdir` just below the line which sets `appArgs to group tweaks to `globalVars` in one place.

josephsl commented:
Hi, oops, should have read the conf file thoroughly. Thanks.

josephsl committed:
Dev docs/Sphinx config: address review comments.

Reviewed by Lukasz Golonka: move globalvars.appDir definition, along with removing duplicate globalVars import statement.
